appveyor/crowdinSync.py

```diff
@@ -0,0 +1,55 @@
+# based on file from https://github.com/jcsteh/osara
```

Comment:
Should appropriate copywrite headers be added?

Reply:
Copyright headers have been added.